astropy/io/fits/verify.py

```diff
@@ -1,4 +1,4 @@
-# Licensed under a 3-clause BSD style license - see PYFITS.rst
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Not sure what is the legally right thing to do when Astropy absorbs something, say, from STScI... Maybe Astropy Project Coordinators ( @taldcroft , @kelle , @astrofrog , and @eteq ) can clarify?